### ISSUE TITLE
SELECT N'' AS NVARCHAR not support

### ISSUE THREAD

**Issue by achariyeak** [ProjectNami/projectnami#462]
wp-db.php Line(2219) _insert_replace_helper
// $formats = implode( ', ', $formats ); 
$formats = implode( ', N', $formats );

**Comment by patrickebates** [comment]
This is in response to the change in last patch https://github.com/ProjectNami/projectnami/pull/422 to force use of Unicode on insert?  So this would be another place in the code that needs updating?

**Comment by achariyeak** [comment]
Yes, need to add (N') here one more place to fully support Unicode in MSSQL.
